android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -580,6 +580,10 @@ private void clearResumePosition() {
     public void onAudioFocusChange(int focusChange) {
         switch (focusChange) {
             case AudioManager.AUDIOFOCUS_LOSS:
+                eventEmitter.audioFocusChanged(false);
+                audioManager.abandonAudioFocus(this);
```

Comment:
If we're going to abandon focus, should we pause automatically? still giving the user the option to stop?

Reply:
This sounds fine, but my idea was to allow the dev to have full control of order of operations, but could be easily done

Reply:
Come to think of it, you're right, this is the right way of _enforcing_ good behavior and will require no changes from the devs. Will add it ASAP,t hanks for the feedback!